test/api/utils/check-security-headers.ts

```diff
@@ -2,10 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 // @flow
-import { typeof Test } from 'supertest';
+import { Test } from 'supertest';
 
 export function checkSecurityHeaders(request: Test) {
-  if (request.called) {
+  if ((request as any).called) {
```

Comment:
I'm a bit surprised about this, are the types for supertest not correct? Should we correct them by overloading the type instead of using any?